plugins/modules/aci_l3out_extepg_to_contract.py

```diff
@@ -0,0 +1,314 @@
+               }
+           }
+       }
+   previous:
+     description: The original configuration from the APIC before the module has started
+     returned: info
+     type: list
+     sample:
+       [
+           {
+               "fvTenant": {
+                   "attributes": {
+                       "descr": "Production",
+                       "dn": "uni/tn-production",
+                       "name": "production",
+                       "nameAlias": "",
+                       "ownerKey": "",
+                       "ownerTag": ""
+                   }
+               }
+           }
+       ]
+   proposed:
+     description: The assembled configuration from the user-provided parameters
+     returned: info
+     type: dict
+     sample:
+       {
+           "fvTenant": {
+               "attributes": {
+                   "descr": "Production environment",
+                   "name": "production"
+               }
+           }
+       }
+   filter_string:
+     description: The filter string used for the request
+     returned: failure or debug
+     type: str
+     sample: ?rsp-prop-include=config-only
+   method:
+     description: The HTTP method used for the request to the APIC
+     returned: failure or debug
+     type: str
+     sample: POST
+   response:
+     description: The HTTP response from the APIC
+     returned: failure or debug
+     type: str
+     sample: OK (30 bytes)
+   status:
+     description: The HTTP status from the APIC
+     returned: failure or debug
+     type: int
+     sample: 200
+   url:
+     description: The HTTP url used for the request to the APIC
+     returned: failure or debug
+     type: str
+     sample: https://10.11.12.13/api/mo/uni/tn-production.json
+   '''
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.network.aci.aci import ACIModule, aci_argument_spec
+
+ACI_CLASS_MAPPING = dict(
+       consumer={
+           'class': 'fvRsCons',
+           'rn': 'rscons-',
+       },
+       provider={
+           'class': 'fvRsProv',
+           'rn': 'rsprov-',
+       },
+)
+
+PROVIDER_MATCH_MAPPING = dict(
+       all='All',
+       at_least_one='AtleastOne',
+       at_most_one='tmostOne',
+       none='None',
+)
+
+def main():
+       argument_spec = aci_argument_spec()
+       argument_spec.update(
+           contract_type=dict(type='str', required=True, choices=['consumer', 'provider']),
+           l3Out=dict(type='str', aliases=['l3Out_name']), 
+           contract=dict(type='str', aliases=['contract_name']),  
+           priority=dict(type='str', choices=['level1', 'level2', 'level3', 'unspecified']),
+           provider_match=dict(type='str', choices=['all', 'at_least_one', 'at_most_one', 'none']),
+           state=dict(type='str', default='present', choices=['absent', 'present', 'query']),
+           tenant=dict(type='str', aliases=['tenant_name']),
+           extepg=dict(type='str',aliases=['externalEpg_name'])  
```

Comment:
this does not match with the alias in the docs. Aliases should be extepg_name and external_epg